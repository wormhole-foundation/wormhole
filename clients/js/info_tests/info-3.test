info contract mainnet ethereum TokenBridge
