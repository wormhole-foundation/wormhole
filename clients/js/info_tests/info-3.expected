0x3ee18B2214AFF97000D974cf647E7C347E8fa585
